web/public/gameData_v1.0-23.json

```diff
@@ -1106,7 +1286,8 @@
                 "power": 4
             },
             "isAlternate": false,
-            "isFicsmas": true
+            "isFicsmas": true,
+            "isPowerGenerator": false
```

Comment:
I don't think this key should exist in the non-power generation recipes. What would it be used for?